04-gas/gas-section-seven/gas-section-seven.html

```diff
@@ -9,39 +9,37 @@
-                class="gas-section-seven-img"
-                src="../../00-overall/05-images/04-gas/gas-section-seven-microwave.png"
-                alt="a cartoon image of a microwave"
-                />
-            </div>
-            <h2 class="gas-section-seven-subtitle">7</h2>
-            <h1 class="gas-section-seven-title">Don't leave appliances on standby</h1>
-            <p class="gas-section-seven-content"> 
-                There are lots of appliances that draw power even when they're on standby. 
-                Figures published by British Gas suggest that UK households waste an 
-                average of £30 a year powering appliances that they're not even using. 
-                Whereas Energy Saving Trust believes you can save even more.
+  <section class="gas-section-seven-container">
 
-                If you make sure that everything is switched off at the wall when 
-                you've finished with it you could save around £40 a year on your 
-                energy bills and 50 kg of CO2. 
+    <section class="gas-section-seven-subcontainer">
+      
+      <div class="gas-section-seven-image-container">
+        <img
+        class="gas-section-seven-image"
+        src="../../00-overall/05-images/04-gas/gas-section-seven-switchusers.png"
+        alt=""
+        />
+      </div>
 
-                This sneaky energy usage is known as vampire power. To combat this, you 
-                could invest in a standby saver, which is a simple solution 
-                to help you navigate the minefield of energy wastage. 
-                Plug all your appliances into a standby saver and then programme 
-                it to respond to a single remote control. It will also automatically cut 
-                the power to an appliance when it realises it has gone into standby mode.
+      <div class="gas-section-seven-copy-container">
+          
+        <div class="gas-section-seven-title-container">
+          <h2 class="gas-section-seven-title"><span class="gas-section-seven-title-span-number">7. </span><span class="gas-section-seven-title-span-title"><span class="bold">
+            Switch gas </span>providers</span></h2>
+        </div>
 
-                Tech can actually help with how to lower a gas bill here, too. You can 
-                buy smart outlets so that you can switch power on and off from 
-                anywhere.
-            </p>
-        </section>
-    <section>
+        <div class="gas-section-seven-paragraph-container">
+          <p class="gas-section-seven-paragraph"> 
+            One of the easiest ways to save money on your gas bill is to ensure that you’re on the most cost-efficient plan available. If you’ve been on the same tariff for a long time, there’s a good chance you’re paying more than you need to. This is because there’s usually quite a gap between the standard variable tariffs most gas companies offer and the cheapest fixed plans available on the market.
```

Comment:
you will see in VSCode the ’ is highlighed yellow, 
you can do a FindAll + ReplaceAll on all to convert it to '
(Ctrl + H)

Reply:
done , thanks , i didnt know its a typo